lib/question.js

```diff
@@ -518,9 +518,8 @@ module.exports = {
                 });
             },
             (callback) => {
-                if (question.grading_method == 'Internal') {
-                    // for Internal grading we call the grading code
-                    questionModule.grade(submission, variant, question, course, (err, ret_courseIssues, ret_data) => {
+                // We always want to call the internet grader in case it can help grade external / manual questions with internal parts
+                questionModule.grade(submission, variant, question, course, (err, ret_courseIssues, ret_data) => {
```

Comment:
I'd be slightly concerned about `grading_jobs_insert` never calling `grading_jobs_insert_internal` when using a grading method of `external/internal`, but it seems like it resolves itself elsewhere. Just a side note.

Reply:
Oh yeah, fortunately the `grading_jobs_insert` file delegates the call: 

![image](https://user-images.githubusercontent.com/7872295/100627795-4c589600-32dc-11eb-9103-7773ad1a7d76.png)